site/docs/v1/tech/apis/entity-management/entities.adoc

```diff
@@ -155,7 +155,7 @@ link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas
 [field]#entityId# [type]#[UUID]# [required]#Required#::
 The Id of the Entity to delete.
 
-include::docs/v1/tech/apis/_x-fusionauth-tenant-id-header-scoped-operation.adoc[]
+include::docs/v1/tech/apis/_x-fusionauth-tenant-id-header-required.adoc[]
```

Comment:
Note that a tenantId can also be provided by an API key, so if you are using a tenant scoped API key, that will satisfy the requirement as well, in which case the header is not required.

Reply:
Reworked.